.github/workflows/version.yml

```diff
@@ -0,0 +1,23 @@
+name: version
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Update SDK Version
+        run: |
+          # Extract the new tag name from the event payload
+          TAG_NAME=${{ github.event.release.tag_name }}
+
+          # Update the sdkVersion variable in deepgram.go file
+          sed -i "s/var sdkVersion string = .*/var sdkVersion string = \"$TAG_NAME\"/" deepgram/deepgram.go
+      - name: Commit changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -m "Update sdkVersion to ${{ github.event.release.tag_name }}" deepgram/deepgram.go
+          git push origin HEAD:${{ github.head_ref }}
```

Comment:
For transparency, this should be done as a PR (programmatically). Pushing directly to `main` loses the traceability of the commit.

Here is an example of programmatically creating and merging a PR:
https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/create-release.sh#L112-L115